nsv13/code/modules/overmap/types/syndicate.dm

```diff
@@ -351,6 +351,16 @@
 	weapon_types[FIRE_MODE_GAUSS] = new /datum/ship_weapon/gauss(src) //AI ships want to be able to use gauss too. I say let them...
 	weapon_types[FIRE_MODE_MISSILE] = new/datum/ship_weapon/missile_launcher(src)
 
+/obj/structure/overmap/syndicate/ai/assault_cruiser/boarding_frigate/elite
+	name = "Special Ops marine frigate"
```

Comment:
```suggestion
	name = "special ops marine frigate"
```

Reply:
Unless "special ops" is the name of a specific group, in which case just use /improper